src/test/resources/Homework18.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="TestNG">
+    <test name="Homework18" preserve-order="false">
+         </classes>
+            <class name="BaseTest"/>
+            <class name="Homework18"/>
+                <methods>
+                    <include name="playSongTest"/>
+                </methods>
+            <class>
+        </classes>
+    </test>
+</suite>
```

Comment:
try fix it. You can refer as below 

![image](https://github.com/Test-Pro-Bootcamp/A47-QA-Automation-Framework/assets/44592718/ba7d576d-d4f7-47a2-b136-51c48b59b224)